File: pset1/cash/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void) 
{
    
    int cent;
    
    do
    {
        float owed_cash = get_float("Enter: ");
        cent = round(owed_cash * 100);
    }
    while(cent <= 0);
    
    int quarters = cent / 25;
    int dimes = (cent % 25) / 10;
    int nickels = ((cent % 25) % 10) / 5;
    int pennies = ((cent % 25) % 10) % 5;

    
    printf("%d\n", quarters + dimes + nickels + pennies);
}

File: pset6/cash.py
while True:
    owed_cash = float(input("Enter: "))
    if owed_cash >= 0:
        break

cent = round(owed_cash * 100)

quarter = cent // 25
dime = (cent % 25) // 10
nickel = ((cent % 25) % 10) // 5
pennie = ((cent % 25) % 10) % 5

output = round(quarter + dime + nickel + pennie)
print(output)